gci/logo.py

```diff
@@ -0,0 +1,27 @@
+import os
+import requests
+
+
+def get_logo():
+    # Obtain the org name from the Travis environment variables
+    org_name = os.environ['TRAVIS_REPO_SLUG'].split('/')[0]

```

Comment:
If the build is done on a local computer, this will fail , because that env var isnt set.

Turns out this is a bad idea, and needs to be re-thought through.

We'll also add another strategy for finding the correct org name, which I have mentioned here:
https://github.com/coala/community/issues/18#issuecomment-349535722
but that isnt part of your task.

Reply:
Should I leave these lines alone then for now?